packages/main/src/plugin/container-registry.ts

```diff
@@ -2405,23 +2404,10 @@ export class ContainerProviderRegistry {
       throw new Error('no running provider for the matching container');
     }
 
-    // generate the name of the exported file
-    // if there are other entries with the same name, let's append (number of occurences)
-    let containerFile = options.name;
-    const regexFile = new RegExp(`${containerFile}\\s+\\(+\\d+\\)$`);
-    const entries = await fs.promises.readdir(options.outputDirectory, { withFileTypes: true });
-    const similarContainers = entries
-      .filter(entry => entry.isFile())
-      .filter(file => file.name === containerFile || regexFile.test(file.name));
-
-    if (similarContainers.length > 0) {
-      containerFile += ` (${similarContainers.length})`;
-    }
-
```

Comment:
if the file already exists we overwrite the content now ?

Reply:
Yes.
In my first implementation i was using the openDialog to make the user select the output folder + there was a textbox where to insert the name of the output file (then i was combining the two and if the file already existed i was adding a `(1)` suffix). 
Now i switched to a saveDialog. So if you try to use the name of an already existing file, it already shows you a warning "do you want to replace the existing file?" (or something like that), so the user is already informed. This is why i removed this part.

Reply:
got it